src/carbonio-files-ui-common/views/components/List.tsx

```diff
@@ -408,9 +433,9 @@ export const List: React.VFC<ListProps> = ({
 
 	const previewSelection = useCallback(() => {
 		const nodeToPreview = find(nodes, (node) => node.id === selectedIDs[0]);
-		const { id, mime_type: mimeType } = nodeToPreview as File;
+		const { id, mime_type: mimeType, type } = nodeToPreview as File;
 		const [$isSupportedByPreview] = isSupportedByPreview(mimeType, 'preview');
-		if ($isSupportedByPreview) {
+		if ($isSupportedByPreview || type === NodeType.Video) {
```

Comment:
Could you remind me why can't we use `permittedSelectionModeActions.includes(Action.Preview)` also here?